guides/Microsoft_Guide.md

```diff
@@ -1,14 +1,65 @@
 # Microsoft Internship Guide
+## General Overview
+Microsoft is one of the pioneers of technology in the area of software and computing, having created Windows OS, Microsoft Office, Azure core and owns Github, Linkdin, Visual studio just to name a few. Microsoft interns mainly use C# and Dotnet and will get the opportunity to work on Microsoft products suc as Azure core or Teams. The bucharest office focuses solely on Azure core and software adjacent to it. 
 
-This guide is currently under construction. Please check back later for detailed information about Microsoft's internship program.
+Microsoft does not provide housing or free food, but does have a very generous relocation package, which you won't be able to spend unless you get a mansion for the summer. 
 
-## Coming Soon
+This guide covers:
+1. How to get a Google STEP / SWE internship
```

Comment:
Wrong here the covers coppied from G, should be 

 Why Choose Microsoft?
 Hiring Process
 Application Submission and screening
 Online Assessment
 Technical Interview (SWE Intern)
 Other info
 
 or maybe join them dunno